core/dbt/include/global_project/macros/db_api/materialized_view/refresh.sql

```diff
@@ -0,0 +1,10 @@
+{% macro db_api__materialized_view__refresh(relation) %}
+    {{ adapter.dispatch('db_api__materialized_view__refresh', 'dbt')(relation) }}
+{% endmacro %}
+
+{% macro default__db_api_refresh_materialized_view(relation) -%}
```

Comment:
Same comment here regarding the naming convention, though the interface macro is correctly named.